.eslintrc.js

```diff
@@ -1,7 +1,9 @@
+"use strict";
```

Comment:
Interestingly, ESLint seems to be linting itself now. I think before it ignored "hidden" files prefixed with a dot, but I guess something changed between v4 and v7.